tests/configs/docker_registry/Dockerfile-direct

```diff
@@ -0,0 +1,2 @@
+FROM python:3.9-slim
+ENTRYPOINT ["python"]
```

Comment:
Why do we need 2 test configs that are identical? The other being `Dockerfile-context`

Reply:
It's just to make sure they are distinguishable in the generated k8s manifests

Reply:
Actually you're right here, simplifying: e0de6ccc87968745ec9fb827fd9a463b